mail_autosubscribe/views/res_partner.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright 2021 Camptocamp (http://www.camptocamp.com).
+    @author Iván Todorovich <ivan.todorovich@gmail.com>
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='email']/parent::div" position="after">
+                <field
+                    name="mail_autosubscribe_ids"
+                    string="In copy of"
```

Comment:
```suggestion
                    string="In copy for"
```
maybe?

Reply:
My english instinct is telling me it's _In copy of_ :man_shrugging: